package.json

```diff
@@ -17,7 +17,21 @@
     "start": "npm install && gulp",
     "watch": "gulp watch",
     "build": "gulp",
-    "deploy": "npm install && composer install --no-dev -o && gulp"
+    "deploy": "npm install && composer install --no-dev -o && gulp",
+    "lint-js": "eslint assets/js",
+    "format-js": "eslint --fix assets/js",
+    "lint": "npm run lint-js",
+    "format": "npm run format-js"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
```

Comment:
Why isn't this just listed as a script?

Reply:
It is not meant to be used as a script (like npm run lint-staged). This configures lint-staged which is meant to be ran on a git hook and will execute eslint for all files prior commit.